apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -201,6 +201,36 @@ static const wifi_manager_ap_crypto_type_e crypto_type_table[] = {
 	WIFI_MANAGER_CRYPTO_UNKNOWN,               /**<  unknown encryption             */
 };
 
+void
+print_wifi_ap_profile(wifi_manager_ap_config_s *config, char *title)
+{
+	printf("====================================\n");
+	if (title) {
+		printf("%s\n", title);
+	}
+	printf("------------------------------------\n");
+	printf("SSID: %s\n", config->ssid);
```

Comment:
Don't you need to check null dereferece for ```config```?

Reply:
I think we don't need check. because it's simple print function.